src/core/meshcop/commissioner.hpp

```diff
@@ -78,6 +79,40 @@ class Commissioner
     ThreadError Stop(void);
 
     /**
+     * This method adds a Joiner entry.
+     *
+     * @param[in]  aExtAddress      A reference to the Joiner's extended address.
```

Comment:
@luwannest, thanks for verifying.

@turon, please avoid making unverified claims.  On the wildcard value, can you provide a technical reason for using constant rather than `NULL`?